posts/2024-11-01-jupyter-book-2-alpha.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Announcing Jupyter Book 2 alpha
+subtitle: Rebuilt from the ground up for prettier PDFs, richer websites, and better tooling.
+date: 2024-11-11
+authors:
+  - name: Angus Hollands
+    orcid: 0000-0003-0788-3814
+    affiliations:
+      - Jupyter Book
+abbreviations:
+  CLI: command-line interface
+license: CC-BY-4.0
+---
+
+Over the last ten months, the Jupyter Book team have been hard at work; Jupyter Book has [become a Jupyter subproject](https://github.com/jupyter/governance/pull/229), and the team[^jb-eb] announced a [plan to release Jupyter Book 2][plan]. This post announces the **alpha release of Jupyter Book 2.0**, which has been re-written from the ground up to use the new MyST-MD engine.
+
+Over the next few months, we will work in preparation for the full release of Jupyter Book 2. Stay tuned for more!
+
+## New Features in `2.0 alpha`
+### Rich Hover Previews
+Try hovering over [this tooltip about tooltips!](https://en.wikipedia.org/wiki/Tooltip). The new MyST book and article themes provide useful hover previews for links to other MyST content, Wikipedia, GitHub issues, and many more.
+
+### Share & Embed Content
+Content from other websites built with the MyST engine can be embedded in your own sites and PDFs:
+% TODO: embed example
+
+### Simple Instant Search
+A new client-side search uses a simple, modern algorithm for fast _local_ search that finds the results that you care about.
+
+% TODO: embed GIF / video
+
+### High Quality PDFs
+
+PDF documents can now be built with Typst, a high-quality typesetting engine that produces readable error messages and beautiful documents.
+
+## Coming Soon in `2.0.0b0`
+
+### Custom Styles & Scrips
+
+Jupyter Book 2 will make it easy to tweak your website styles, and add new website behaviors.
+
+### Generate Markdown from Code Cells
+
+The MyST engine is on-track to support the inclusion of references and other markup features generated by code cells.
+
+### Control Cell Visibility with Tags
+
+In the beta release, Jupyter Book 2 will once-again be able to show and hide content according to cell tags.
+
+## Jupyter Book 2 vs MyST-MD
+
+% TODO testimonial from JB 1->2 users
```

Comment:
Don't worry about this testimonial stuff, too much overhead for an alpha blog post